src/NuGet.Clients/VisualStudio/Extensibility/CredentialRequestType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// The type of credential request being made.
+    /// </summary>
+    public enum CredentialRequestType
```

Comment:
What do you mean by a normal request? Currently we only prompt when reacting to a 401, 403, or 407.